### DISCUSSION TITLE
Overlong Text Wrapping

### DISCUSSION THREAD

**Discussion by syc795** [eclipse-birt/birt#1488] [Q&A] [answered]
I created a simple table with a field that might be long, and I expected the PDF to be wrapped, but I didn't find this configuration.
Rptdesign:
![image](https://github.com/eclipse-birt/birt/assets/36059674/15fbe55f-ae14-410e-b3f3-78ac08e18838)
DataSource:
```xml
<dataSource>
	<data>
		<serviceNumber>1111111111111111111122222222222222222222222223333333333333333333333333333344444444444444444444444444455555555555555555555555555</serviceNumber>
		<serviceType>3</serviceType>
	</data>
	<data>
		<serviceNumber>3</serviceNumber>
		<serviceType>4</serviceType>
	</data>
</dataSource>
```
PDF Preview:
![image](https://github.com/eclipse-birt/birt/assets/36059674/8503ed13-52b4-406a-a596-8045d07ce9e6)

**Comment by hvbtup** [comment, marked as answer]
The preview doesn't support this (or at least I don't know how to configure it), but if you are generating a report programmatically using e.g. `RunAndRenderTask`, then you can set the `RenderOption.PDF_WORDBREAK` (= "pdfRenderOption.wordBreak") to True.
